### PR TITLE
Typing improvement

### PR DESCRIPTION
**Pull request by princesinha19**
## Description

Improving typing of modules.

## Type of change
- [x] Enhancement

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no warnings.
- [x] I have updated or added types for all modules I've changed
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran ```npm run test``` in the root folder with success and extended the tests if necessary.
- [x] I ran ```npm run build``` in the root folder and tested it in the browser and with node.
- [x] I ran ```npm run dtslint``` in the root folder and tested that all my types are correct
- [x] I have tested my code on an ethereum test network.

### CODE REVIEW

**Comment by princesinha19** [comment]
@joshstevens19, @nivida can you please review this?

**Comment by princesinha19** [comment]
@nivida Can you please merge this?

File: packages/web3-eth/types/index.d.ts [hunk 1]
```diff
@@ -242,3 +242,9 @@ export interface StorageProof {
     value: string;
     proof: string[];
 }
+
+export interface PendingTransactions extends Transaction {
```
Comment (nivida) [inline packages/web3-eth/types/index.d.ts]: I would rename ``PendingTransactions`` to ``PendingTransaction``

**Review by nivida — CHANGES_REQUESTED** [review]

**Comment by princesinha19** [comment]
Could you please merge this @nivida?

**Comment by princesinha19** [comment]
@nivida Any specific reason behind closing this PR?

**Comment by joshstevens19** [comment]
I think your going to have to do this against the 2.0 alpha branch now due to the changes of a stable 1.0. So I’m assuming he closed it but forgot to say way.. maybe you could redo this against the 2.0 alpha  then we can get it merged 👍

**Pull request closed without merging**